ct-app/core/components/node_helper.py

```
@@ -1,11 +1,15 @@
+from re import U
```

Ruff (0.8.2)

`re.U` imported but unused

Remove unused import: `re.U`

(F401)